presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -100,6 +100,10 @@ Date and Time Functions
 
     Returns ``timestamp`` as a UNIX timestamp.
 
+.. function:: to_milliseconds(interval day to second) -> bigint
+
+    Returns the number of milliseconds in the interval.
```

Comment:
I would say "converts the interval" as it's slightly ambiguous if this is simply extracting the milliseconds component.